### DISCUSSION TITLE
For Automatic Periodic Boundary Detection

### DISCUSSION THREAD

**Discussion by snugook1108** [idaholab/moose#21278] [Q&A General] [answered]
## Bug Description
For Automatic Periodic Boundary Detection, can it use the AuxVariables? It is only for the Variables System.

**Comment by hugary1995** [comment]
This isn't a bug. You should post in the discussions page.

**Comment by GiudGiud** [comment, marked as answer]
There are no periodic BCs for AuxVariables because we typically dont set boundary conditions to auxvariables. They are not solved for using a nonlinear solver, they are computed after the fact based on other fields/quantities in a straightforward process using AuxKernels
